.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ["bug", "triage"]
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
I'd leave out the example. You have to delete this every time. Will become annoying. Maybe we could add an example of how to report an issue to the `CONTRIBUTING.md`. That might be nice.

Reply:
I do not think people read the fine manuals first.

I agree with 'always delete' but seen many projects too ... let's see what happens?